lib/simp/cli/commands/bootstrap.rb

```diff
@@ -251,23 +229,18 @@ def self.run(args = [])
         Process.kill('KILL',pserver_pid)
       end
     rescue Exception => e
-      puts e
-      puts "The Puppet Server process running on port 8150 could not be killed. Please check your configuration!"
+      say e
+      say "> <%= color('The Puppet Server process running on port 8150 could not be killed. Please check your configuration!', :magenta) %>"
```

Comment:
I don't believe so. The agent will be configured to check in to whatever port was configured in simp config. After reviewing this code, however, it should be noted that if the user intends and configures puppetserver to run on 8150, it will kill it anyways. In fact, if *anything* is running on 8150, it'll kill it.  I'll patch it.